cpp-package/include/mxnet-cpp/monitor.h

```diff
@@ -70,8 +70,9 @@ class Monitor {
   /*!
   * \brief install callback to executor. Supports installing to multiple executors.
   * \param exe The executor to install to.
+  * \param monitor_all If true, monitor both input and output, otherwise monitor output only.
   */
-  void install(Executor *exe);
+  void install(Executor *exe, bool monitor_all = false);
```

Comment:
`\param monitor_all If true, monitor both input and output, otherwise monitor output only.`
If it's not clear enough, please leave your questions here and I can add more description.